packages/lesswrong/components/posts/PostsAnnualReviewMarketTag.tsx

```diff
@@ -0,0 +1,83 @@
+import { AnnualReviewMarketInfo } from '../../lib/annualReviewMarkets';
+import { useSingle } from '../../lib/crud/withSingle';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import React from 'react';
+import { useHover } from '../common/withHover';
+
+const styles = (theme: ThemeType) => ({
+  expectedWinner: {
+    color: theme.palette.review.winner,
+    border: '1px solid',
+    borderColor: theme.palette.review.winner,
+    borderRadius: '4px',
+    fontFamily: theme.typography.fontFamily,
+    width: 'fit-content',
+    display: 'inline-block',
+    padding: '5px',
+    // marginTop: 8,
+    // marginBottom: 8,
+    boxSizing: 'border-box',
+  },
+  preview: {
+    maxWidth: 400,
+  },
+});
+
+const PostsAnnualReviewMarketTag = ({ post, annualReviewMarketInfo, classes }: {
+  post: PostsWithNavigation | PostsWithNavigationAndRevision | PostsList,
+  annualReviewMarketInfo: AnnualReviewMarketInfo | null,
+  classes: ClassesType<typeof styles>,
+}) => {
+
+  const { CommentsNode, LWPopper } = Components;
+  const { anchorEl, hover, eventHandlers } = useHover();
+
+  const annualReviewMarketCommentId = post.annualReviewMarketCommentId
+
+  const { document: comment } = useSingle({
+    documentId: annualReviewMarketCommentId,
+    collectionName: "Comments",
+    fragmentName: "CommentsList",
+  });
+
+  if (!annualReviewMarketInfo) {
+    return <div></div>;
+  }
+
+  const decimalPlaces = 0;
+  return <span>
+    <div className={classes.expectedWinner} {...eventHandlers}>
+      {annualReviewMarketInfo?.year} Top Fifty: {parseFloat((annualReviewMarketInfo?.probability * 100).toFixed(decimalPlaces))}%
```

Comment:
Shouldn't need the null-safe access given you're doing an early return if it doesn't exist a few lines above